src/Installer/pkg/dotnet-sdk.proj

```diff
@@ -0,0 +1,129 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+  <PropertyGroup>
+    <SkipBuild Condition="'$(BuildSdkDeb)' != 'true' and '$(IsRPMBasedDistro)' != 'true'">true</SkipBuild>
+    <TargetFramework>$(SdkTargetFramework)</TargetFramework>
+    <RuntimeIdentifier>$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
+    <GenerateInstallers>true</GenerateInstallers>
+    <BuildDebPackage Condition="'$(BuildSdkDeb)' == 'true'">true</BuildDebPackage>
+    <BuildRpmPackage Condition="'$(IsRPMBasedDistro)' == 'true'">true</BuildRpmPackage>
+    <UseArcadeRpmTooling>true</UseArcadeRpmTooling>
+    <ProductBrandPrefix>Microsoft .NET</ProductBrandPrefix>
+    <LicenseFile>$(RepoRoot)LICENSE.TXT</LicenseFile>
+    <InstallerName>dotnet-sdk</InstallerName>
+    <PackageBrandNameSuffix>SDK</PackageBrandNameSuffix>
+    <!--
+      Installer infra requires MajorVersion and MinorVersion, which triggers reevaluation of Arcade's
+      VersionPrefix property. This requires PatchVersion to be set in order to produce the correct SDK version.
+    -->
+    <MajorVersion>$(VersionMajor)</MajorVersion>
+    <MinorVersion>$(VersionMinor)</MinorVersion>
+    <PatchVersion>$(VersionPrefix.Split('.')[2])</PatchVersion>
+    <!--
+      All supported installer runtime identifiers should be specified here.
+      New arcade infra is only used for Linux installers, at the moment.
+    -->
+    <InstallerRuntimeIdentifiers>linux-$(Architecture)</InstallerRuntimeIdentifiers>
+    <IsShippingPackage>true</IsShippingPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\redist-installer\redist-installer.csproj" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" />
+  </ItemGroup>
+
+  <Import Project="$(RepoRoot)src\Tasks\sdk-tasks\sdk-tasks.InTree.targets" />
+  <Import Project="..\redist-installer\targets\LinuxNativeInstallerDependencyVersions.targets" />
```

Comment:
This targets file is now only used by this project, it's unused in redist-installer.csproj. Please inline the target into here and delete the file.

Reply:
Yeah, good point - will fix this.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/45047/commits/dfb7abc63d8eafcef9ae7b13ccf36975aa8162bb